creator-node/src/README.md

```diff
@@ -0,0 +1,37 @@
+All `creator-node` code should adhere to the below structure to ensure separation of concerns.
```

Comment:
big pour for whoever renames this dir to content-node 🍺

Reply:
oof yeah